recipe/meta.yaml

```
@@ -44,17 +44,19 @@ test:
   requires:
     - boto3 >=1.17.46
     - composeml >=0.3.0
-    - fastparquet==0.5.0
-    - graphviz !=2.47.2
+    - graphviz >=0.8.4
```

I think we will need python graphviz too:
```suggestion
    - graphviz !=2.47.2
    - python-graphviz >=0.8.4
```